homeassistant/components/supla/__init__.py

```diff
@@ -130,6 +140,23 @@ def name(self) -> Optional[str]:
         """Return the name of the device."""
         return self.channel_data["caption"]
 
+    @property
+    def hidden(self) -> bool:
+        """Return True if the entity should be hidden from UIs."""
+        if self.channel_data is None:
+            return False
+        return self.channel_data.get("hidden")
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        if self.channel_data is None:
```

Comment:
Please combine the two if statements since they return the same value.

Reply:
I'm not sure what do you mean by that.

Reply:
Oh sorry read that wrong.